lib/engine/config/tile.rb

```diff
@@ -487,7 +496,9 @@ module Tile
         '469' => 'path=a:0,b:1,track:narrow',
       }.freeze
 
-      COLORS = %i[white yellow green brown gray blue greenbrown browngray none red].freeze
+      PURPLE = {}.freeze
```

Comment:
Intention behind this?

Reply:
without changing either the COLORS or adding PURPLE (or both, did not try just one actually), I recevied an error